kernel/src/lib.rs

```diff
@@ -585,11 +585,18 @@ pub trait ParquetHandler: AsAny {
     /// the columns requested by physical schema . The ParquetHandler _must_ return exactly the
     /// columns specified in `physical_schema`, and they _must_ be in schema order.
     ///
+    /// If a [`StructField`] in the  physical schema contains a field id, then the id is
+    /// used to resolve the parquet column. The field id, if present, is specified in the
```

Comment:
I don't think it's super clear what "resolve" means here. Maybe we should have a small example of how this is supposed to work?

Reply:
Fixed up the docs, PTAL